### PR TITLE
Rwc2018/challenge restaurant2

### PR DESCRIPTION
**Pull request by MatthijsBurgh**

### CODE REVIEW

File: challenge_restaurant/src/challenge_restaurant/store_waypoint.py [hunk 1]
```diff
@@ -22,6 +20,46 @@ def _get_area(convex_hull):
     return area
 
 
+class AutomaticSideDetection2(smach.State):
```
Comment (LoyVanBeek) [inline challenge_restaurant/src/challenge_restaurant/store_waypoint.py]: A better name would be helpfull :-)

**Comment by MatthijsBurgh** [comment]
Fixes #598

**Comment by MatthijsBurgh** [comment]
Fixes #600

File: challenge_restaurant/src/challenge_restaurant/store_waypoint.py (hunk 1 above)
Comment (MatthijsBurgh) [reply challenge_restaurant/src/challenge_restaurant/store_waypoint.py]: Better docstring would be helpfull

**Pull request merged**